jobs/v3/quickstart/main.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START quick_start]
+
+// This is a quickstart sample of using the Google Cloud Job Discovery API.
+package main
+
+import (
+	"os"
+	"fmt"
+	"log"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	talent "google.golang.org/api/jobs/v3"
+)
+
+func main() {
+	// Authorize the client using Application Default Credentials.
+	// See https://g.co/dv/identity/protocols/application-default-credentials
+	ctx := context.Background()
+	client, err := google.DefaultClient(ctx, talent.CloudPlatformScope)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+
+	// Create the jobs service client.
+	cloudTalentSolutionSerivce, err := talent.New(client)
```

Comment:
Would prefer a shorter name here. `js` is used below for jobs service. Should this be `ts`?